### PR TITLE
API to create ticket

### PR DESCRIPTION
**Pull request by sk1384**

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -32,6 +32,8 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  # config.extend ApiHeader
```
Comment (Selectus2) [inline spec/rails_helper.rb:35]: remove commented code

File: config/locales/en.yml [hunk 2]
```diff
@@ -36,7 +36,12 @@ en:
   login:
     success: "Logged in successfully"
     failure: "Invalid Credentials"
-  hello: "Hello world"
+
+  tickets:
```
Comment (Selectus2) [inline config/locales/en.yml:40]: use either one of ticktes or ticket

File: app/services/tickets/v1/create.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    attr_accessor :title, :description, :ticket, :category_id, :department_id,
+                  :current_user, :priority, :resolver_id, :requester_id, :ticket_type
+
+    def initialize(params, user)
+      @current_user = user
+      @title = params[:title]
+      @description = params[:description]
+      @category_id = params[:category_id]
+      @department_id = params[:department_id]
+      @resolver_id = params[:resolver_id]
```
Comment (Selectus2) [inline app/services/tickets/v1/create.rb:12]: resolver is assigned based on department right ?

File: app/services/tickets/v1/create.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      @ticket_type = params[:ticket_type]
+    end
+
+    def call
+      set_values && save_ticket
+    end
+
+    def set_values
```
Comment (Selectus2) [inline app/services/tickets/v1/create.rb:20]: set which values

**Review by Selectus2 — CHANGES_REQUESTED** [review]

File: app/services/tickets/v1/create.rb (hunk 4 above)
Comment (sk1384) [reply app/services/tickets/v1/create.rb]: Done

File: app/services/tickets/v1/create.rb (hunk 3 above)
Comment (sk1384) [reply app/services/tickets/v1/create.rb]: Done

**Review by Selectus2 — APPROVED** [review]
LGTM. Please consider sending specific attribute level errors in create api failure response

**Pull request merged**